### PR TITLE
add className support for Dots

### PR DESCRIPTION
**Pull request by mikeechen**
- [x] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [x] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

-Support for issue #396

### CODE REVIEW

File: src/components/CarouselDots.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ class CarouselDots extends Component {
     value: PropTypes.number,
     onChange: PropTypes.func,
     rtl: PropTypes.bool,
+    className: PropTypes.string,
```
Comment (piotr-s-brainhub) [inline src/components/CarouselDots.js:13]: could you add it to the README?

Comment (mikeechen) [reply src/components/CarouselDots.js:13]: Will do!

Comment (mikeechen) [reply src/components/CarouselDots.js:13]: @piotr-s-brainhub let me know if that readme works and what else I might need to change?

Comment (mikeechen) [reply src/components/CarouselDots.js:13]: I'll probably need some help on making the build work too....

**Comment by piotr-s-brainhub** [comment]
@mikeechen 

Thanks.
I opened https://github.com/brainhubeu/react-carousel/pull/398 instead because only the main repo provides an ability to build a testing environment.

**Comment by mikeechen** [comment]
> @mikeechen
> 
> Thanks.
> I opened #398 instead because only the main repo provides an ability to build a testing environment.

Thank you!

**Pull request merged**